NitroxClient/GameLogic/Entities.cs

```diff
@@ -174,15 +161,10 @@ private void AddPendingParentEntity(Entity entity)
             pendingEntities.Add(entity);
         }
 
-        public bool WasSpawnedByServer(NitroxId id)
-        {
-            return alreadySpawnedIds.Contains(id);
-        }
+        private Int3 ToInt3(NitroxModel.DataStructures.Int3 int3) => new Int3(int3.X, int3.Y, int3.Z);
```

Comment:
This should be transferred to [DataExtensions](https://www.github.com/SubnauticaNitrox/Nitrox/tree/master/NitroxModel-Subnautica%2FDataStructures%2FDataExtensions.cs)